platform/barefoot/sonic-platform-modules-imt/setup.py

```diff
@@ -0,0 +1,13 @@
+from setuptools import setup
+
+setup(
+    name='sonic-platform',
+    version='1.0',
+    description='SONiC platform API implementation',
+    author_email='support@interfacemasters.com',
+    packages=[
+        'sonic_platform',
+        'sonic_platform/pltfm_mgr_rpc',
+    ],
+    package_data = {'sonic_platform':['logging.conf']},
```

Comment:
Seems like we don't have 'logging.conf' file that can be added to our package.

Reply:
fixed